### ISSUE TITLE
FSE: Unified keyboard shortcuts for sidebars

### ISSUE THREAD

**Issue by critterverse** [WordPress/gutenberg#30294]
As keyboard shortcuts are in the process of being implemented in the Full Site Editor as a part of #[21245](https://github.com/WordPress/gutenberg/issues/21245), it might be a good time to discuss potential keyboard shortcuts for the additional sidebars that have been added. 

The sidebars that can be toggled from the top toolbar, from left to right, are:

- Full Site Editor navigation
- Inserter
- List View (existing shortcut: ^⌥O)
- Settings (existing shortcut: ⇧⌘,)
- Global Styles

_*Please comment below if I'm missing any existing sidebar shortcuts_

I can imagine wanting to unify the open/closing behavior of all the sidebars through similar, easy-to-remember shortcuts. For example, each of the sidebars could be given some variation of the existing shortcut for toggling Settings or List View.

A different approach could be to make it easy to cycle through all the possible sidebar configurations via a shortcut similar to the one used for jumping to the next part of the editor: ^⌥N. Whereas the current shortcut cycles through the open sections of the editor, a new shortcut could cycle through each of the possible sidebar configurations, opening each sidebar one at a time (including an "all closed" state).

Would love to discuss which approach could best help improve usability in the FSE.

**Comment by critterverse** [comment]
Closing this issue as navigation in the site editor nows behaves differently.
